pypxlib/__init__.py

```diff
@@ -2,6 +2,7 @@
 from datetime import date, time
 from pypxlib.pxlib_ctypes import *
 
+import os.path
```

Comment:
I would prefer this to be `from os.path import isfile`, and to have it between lines 2 and 3 above (so the `from ... import`s are sorted alphabetically).